server/leafnode.go

```diff
@@ -1764,6 +1764,13 @@ func (c *client) processLeafNodeConnect(s *Server, arg []byte, lang string) erro
 		return err
 	}
 
+	// Reject a cluster that contains spaces or line breaks.
+	if proto.Cluster != _EMPTY_ && strings.ContainsAny(proto.Cluster, " \n") {
```

Comment:
`CONNECT_ARG` in the parser is also sensitive to `\r` and `OP_CONNECT` is sensitive to `\t` so let's include those too.

Reply:
thanks, now added